tensorflow/python/ops/image_ops_test.py

```diff
@@ -4071,6 +4071,14 @@ def testShape(self):
         self.assertEqual(image.get_shape().as_list(),
                          [None, None, channels or None])
 
+  def testEmptyTensor(self):
+    with self.cached_session(use_gpu=True) as sess:
+      image = array_ops.reshape(
+          constant_op.constant([], dtypes.uint8), [0, 0, 3])
+      with self.assertRaisesOpError(r"image data should not be empty"):
+        png = image_ops.encode_png(image)
```

Comment:
Hi @yongtang 
Failing with similar error internally , can you please check once,
`Traceback (most recent call last):
  /unittest/case.py", line 59, in testPartExecutor
    yield
  /unittest/case.py", line 605, in run
    testMethod()
  File "/google3/runfiles/google3/third_party/tensorflow/python/ops/image_ops_test.py", line 4079, in testEmptyTensor
    png = image_ops.encode_png(image)
  File "contextlib.py", line 88, in __exit__
    next(self.gen)
  File "google3/runfiles/google3/third_party/tensorflow/python/framework/test_util.py", line 2819, in assertRaisesWithPredicateMatch
    (str(type(e)), str(e)))
AssertionError: Exception of type <class 'AssertionError'>: OpError not raised`